### ISSUE TITLE
Bug: Loading Movenet gives an undefined error for TFJS

### ISSUE THREAD

**Issue by MarioCavero** [tensorflow/hub#891]
### What happened?

I have been using movenet without any problems. I have been following the guide. I posted an [issue](https://github.com/tensorflow/tfjs/issues/7930) in TFJS and it was tried following another methodology, which sometimes gives and sometimes does not give errors.

Does this happen sometimes? Is it possible to do it always locally if the model is downloaded, to avoid "no internet" issues?
If so:
I also loaded the movenet model for TFJS but there are no `.predict` or `.estimatePoses` methods. Is it possible to use the movenet pretrained model as a loading graph model, to do the same inferences as creating a detector from the model as the guide says, and doing `.estimatePoses`?

### Relevant code

```python
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"></script>
  <script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/pose-detection"></script>

    await tf.setBackend('webgl');
    await tf.ready();

    const detectorConfig = {
      modelType: poseDetection.movenet.modelType.SINGLEPOSE_LIGHTNING,
    };
    const detector = await poseDetection.createDetector(poseDetection.SupportedModels.MoveNet, detectorConfig);
```

### Relevant log output

```shell
pose-detection:17 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'loadGraphModel')
    at pose-detection:17:56068
    at pose-detection:17:2398
    at Object.next (pose-detection:17:2503)
    at pose-detection:17:1440
    at new Promise (<anonymous>)
    at u (pose-detection:17:1185)
    at Xt (pose-detection:17:55433)
    at Object.<anonymous> (pose-detection:17:71747)
    at pose-detection:17:2398
    at Object.next (pose-detection:17:2503)

And going to the source of the error in console:

case 2:
                    return a = void 0,
                    "SinglePose.Lightning" === e.modelType ? a = "https://tfhub.dev/google/tfjs-model/movenet/singlepose/lightning/4" : "SinglePose.Thunder" === e.modelType ? a = "https://tfhub.dev/google/tfjs-model/movenet/singlepose/thunder/4" : "MultiPose.Lightning" === e.modelType && (a = "https://tfhub.dev/google/tfjs-model/movenet/multipose/lightning/1"),
                    [4, i.loadGraphModel(a, {
                        fromTFHub: o
 })];
```

### tensorflow_hub Version

? Following guide, installed for npm. Worked fine until today. 

### TensorFlow Version

2.8 (latest stable release)

### Python Version

3.x

### OS

Linux

**Comment by herdiansah** [comment]
I have the same issue, My application suddenly has the same error "Uncaught TypeError: Cannot read properties of undefined (reading 'loadGraphModel') pose-detection:17"

**I have try using simple code for multipose lightning testing:**
const modelUrl = 'https://tfhub.dev/google/tfjs-model/movenet/multipose/lightning/1';
const model = await tf.loadGraphModel(modelUrl, {fromTFHub: true});
console.log(model);

**I have this error:**
Uncaught TypeError: Cannot read properties of undefined (reading 'loadGraphModel')
    at pose-detection:17:56068
    at pose-detection:17:2398
    at Object.next (pose-detection:17:2503)
    at pose-detection:17:1440
    at new Promise (<anonymous>)
    at u (pose-detection:17:1185)
    at Xt (pose-detection:17:55433)
    at Object.<anonymous> (pose-detection:17:71747)
    at pose-detection:17:2398
    at Object.next (pose-detection:17:2503)

above simple code used to be working fine.

**Comment by singhniraj08** [comment]
@MarioCavero,

Referring the TFJS issue filled by you, looks like the issue was from TFJS side and issue is happening with latest versions of `@tensorflow-models/pose-detection@2.1.1` and `@tensorflow-models/pose-detection@2.1.2` and it's working as expected with this version `@tensorflow-models/pose-detection@2.1.0`. The issue should be fixed with `npm version@2.1.3`.

Kindly let us know if this issue is resolved for you. Thank you!

**Comment by herdiansah** [comment]
I have tested the updated pose-detection and now my application working fine.

thanks
